### ISSUE TITLE
[Bug]: LDAP User and Group Backend - Groups are not shown in Settings -> User, but groups are available on CLI and in other dialogs

### ISSUE THREAD

**Issue by nagmat84** [nextcloud/server#42474]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

I have set up LDAP User and Group Backend for users and groups. LDAP users, groups and group memberships seem to work, but when I navigate to "Settings -> Users" neither the groups are shown nor does the page list the group membership of the LDAP users.

Expected behavior: Settings -> User should properly show LDAP groups and group memberships for LDAP users
Actual behavior: Settings -> Users only shows LDAP users, but neither shows LDAP groups nor group memberhips

When I press the button "Verify settings and count groups" during setup, Nextcloud correctly reports the number of groups. When I run `occ group:list` on the CLI, `occ` correctly reports all LDAP groups. `occ group:info` also works for an LDAP group. When I run `occ:user:info <some LDAP UUID>` on the CLI, `occ` correctly reports LDAP group memberships. When someone tries to share something (file/directory/calendar/other stuff) and starts typing the name of the entity to share with, auto-completion also includes LDAP groups. It is possible to share stuff with LDAP groups and access rights are granted to users as expected.

Screenshot of the Settings -> User page (**Note: Groups and group memberships are _erroneously NOT_ shown**):

![SettingsUser](https://github.com/nextcloud/server/assets/7428426/53bc8599-e105-43bf-9402-146bcde7bed9)

Screenshot of CLI commands:

![CLI](https://github.com/nextcloud/server/assets/7428426/adf9f390-8b1b-4374-bdf0-4a1ad981c53a)

Screenshot of auto-completion during sharing (LDAP groups actually work and are found):

![SharingAutoCompletion](https://github.com/nextcloud/server/assets/7428426/9769868f-7997-4885-b736-47f5e1fae3cc)

### Steps to reproduce

1. Configure LDAP User and Group Backend
2. Populate your LDAP DIT with some users, groups and group memberships
3. Go to Settings -> User in Nextcloud
4. Groups and group memberships are not shown

### Expected behavior

Groups and group memberships should be shown.

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

28

### Operating system

Other

### PHP engine version

PHP 8.2

### Web server

Apache (supported)

### Database engine version

PostgreSQL

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.famna.de",
            "cloud.mhnnet.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "28.0.1.1",
        "overwrite.cli.url": "https:\/\/cloud.famna.de",
        "htaccess.RewriteBase": "\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0
        },
        "mail_smtpmode": "sendmail",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "maintenance": false,
        "skeletondirectory": "",
        "templatedirectory": "",
        "default_language": "de",
        "default_locale": "de_DE",
        "default_phone_region": "DE",
        "default_timezone": "Europe\/Berlin",
        "enabledPreviewProviders": [
            "OC\\Preview\\BMP",
            "OC\\Preview\\GIF",
            "OC\\Preview\\HEIC",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\Krita",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\MSOffice2003",
            "OC\\Preview\\MSOffice2007",
            "OC\\Preview\\MSOfficeDoc",
            "OC\\Preview\\Movie",
            "OC\\Preview\\MP3",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\PDF",
            "OC\\Preview\\PNG",
            "OC\\Preview\\SVG",
            "OC\\Preview\\TIFF",
            "OC\\Preview\\TXT",
            "OC\\Preview\\WebP",
            "OC\\Preview\\XBitmap"
        ],
        "allow_user_to_change_display_name": false
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.20.0
  - bruteforcesettings: 2.8.0
  - calendar: 4.6.1
  - calendar_resource_management: 0.6.0
  - cloud_federation_api: 1.11.0
  - contacts: 5.5.0
  - dashboard: 7.8.0
  - dav: 1.29.1
  - event_update_notification: 2.3.0
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_accesscontrol: 1.18.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - groupfolders: 16.0.1
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - mail: 3.5.0
  - nextcloud_announcements: 1.17.0
  - notifications: 2.16.0
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - previewgenerator: 5.4.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - support: 1.11.0
  - tasks: 0.15.0
  - text: 3.9.1
  - theming: 2.3.0
  - twofactor_backupcodes: 1.17.0
  - updatenotification: 1.18.0
  - user_ldap: 1.19.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - workflowengine: 2.10.0
Disabled:
  - admin_audit: 1.18.0
  - circles: 28.0.0-dev (installed 28.0.0-dev)
  - comments: 1.18.0 (installed 1.18.0)
  - contactsinteraction: 1.9.0 (installed 1.9.0)
  - encryption: 2.16.0
  - files_external: 1.20.0
  - files_pdfviewer: 2.9.0 (installed 2.9.0)
  - files_reminders: 1.1.0 (installed 1.1.0)
  - firstrunwizard: 2.17.0 (installed 2.17.0)
  - photos: 2.4.0 (installed 2.4.0)
  - privacy: 1.12.0 (installed 1.12.0)
  - survey_client: 1.16.0 (installed 1.16.0)
  - suspicious_login: 6.0.0
  - systemtags: 1.18.0 (installed 1.18.0)
  - twofactor_totp: 10.0.0-beta.2
  - weather_status: 1.8.0 (installed 1.8.0)
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"t5kiCaD77yGqMApQxk35","level":3,"time":"2023-12-22T17:10:01+00:00","remoteAddr":"79.222.238.167","user":"7745f1ae-d552-4a8a-bbd2-1a1fd709edbf","app":"no app in context","method":"PUT","url":"/ocs/v2.php/apps/user_status/api/v1/heartbeat?format=json","message":"Exception thrown: OC\\DB\\Exceptions\\DbalException","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/119.0","version":"28.0.1.1","exception":{"Exception":"OC\\DB\\Exceptions\\DbalException","Message":"An exception occurred while executing a query: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"user_status_uid_ix\"\nDETAIL:  Key (user_id)=(7745f1ae-d552-4a8a-bbd2-1a1fd709edbf) already exists.","Code":7,"Trace":[{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":328,"function":"wrap","class":"OC\\DB\\Exceptions\\DbalException","type":"::"},{"file":"/var/www/famna.de/nextcloud/lib/public/AppFramework/Db/QBMapper.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Listener/UserLiveStatusListener.php","line":113,"function":"insert","class":"OCP\\AppFramework\\Db\\QBMapper","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/ServiceEventListener.php","line":86,"function":"handle","class":"OCA\\UserStatus\\Listener\\UserLiveStatusListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":230,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":59,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":94,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":106,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Controller/HeartbeatController.php","line":100,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"heartbeat","class":"OCA\\UserStatus\\Controller\\HeartbeatController","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/famna.de/nextcloud/ocs/v1.php","line":65,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/famna.de/nextcloud/ocs/v2.php","line":23,"args":["/var/www/famna.de/nextcloud/ocs/v1.php"],"function":"require_once"}],"File":"/var/www/famna.de/nextcloud/lib/private/DB/Exceptions/DbalException.php","Line":71,"Previous":{"Exception":"Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException","Message":"An exception occurred while executing a query: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"user_status_uid_ix\"\nDETAIL:  Key (user_id)=(7745f1ae-d552-4a8a-bbd2-1a1fd709edbf) already exists.","Code":7,"Trace":[{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Connection.php","line":1938,"function":"convert","class":"Doctrine\\DBAL\\Driver\\API\\PostgreSQL\\ExceptionConverter","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Connection.php","line":1880,"function":"handleDriverException","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Connection.php","line":1208,"function":"convertExceptionDuringQuery","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/public/AppFramework/Db/QBMapper.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Listener/UserLiveStatusListener.php","line":113,"function":"insert","class":"OCP\\AppFramework\\Db\\QBMapper","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/ServiceEventListener.php","line":86,"function":"handle","class":"OCA\\UserStatus\\Listener\\UserLiveStatusListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":230,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":59,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":94,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":106,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Controller/HeartbeatController.php","line":100,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"heartbeat","class":"OCA\\UserStatus\\Controller\\HeartbeatController","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/famna.de/nextcloud/ocs/v1.php","line":65,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/famna.de/nextcloud/ocs/v2.php","line":23,"args":["/var/www/famna.de/nextcloud/ocs/v1.php"],"function":"require_once"}],"File":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php","Line":53,"Previous":{"Exception":"Doctrine\\DBAL\\Driver\\PDO\\Exception","Message":"SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"user_status_uid_ix\"\nDETAIL:  Key (user_id)=(7745f1ae-d552-4a8a-bbd2-1a1fd709edbf) already exists.","Code":7,"Trace":[{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":132,"function":"new","class":"Doctrine\\DBAL\\Driver\\PDO\\Exception","type":"::"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Connection.php","line":1202,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/public/AppFramework/Db/QBMapper.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Listener/UserLiveStatusListener.php","line":113,"function":"insert","class":"OCP\\AppFramework\\Db\\QBMapper","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/ServiceEventListener.php","line":86,"function":"handle","class":"OCA\\UserStatus\\Listener\\UserLiveStatusListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":230,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":59,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":94,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":106,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Controller/HeartbeatController.php","line":100,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"heartbeat","class":"OCA\\UserStatus\\Controller\\HeartbeatController","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/famna.de/nextcloud/ocs/v1.php","line":65,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/famna.de/nextcloud/ocs/v2.php","line":23,"args":["/var/www/famna.de/nextcloud/ocs/v1.php"],"function":"require_once"}],"File":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Exception.php","Line":28,"Previous":{"Exception":"PDOException","Message":"SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"user_status_uid_ix\"\nDETAIL:  Key (user_id)=(7745f1ae-d552-4a8a-bbd2-1a1fd709edbf) already exists.","Code":"23505","Trace":[{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":130,"function":"execute","class":"PDOStatement","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Connection.php","line":1202,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/public/AppFramework/Db/QBMapper.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Listener/UserLiveStatusListener.php","line":113,"function":"insert","class":"OCP\\AppFramework\\Db\\QBMapper","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/ServiceEventListener.php","line":86,"function":"handle","class":"OCA\\UserStatus\\Listener\\UserLiveStatusListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":230,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/var/www/famna.de/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":59,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":94,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":106,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/apps/user_status/lib/Controller/HeartbeatController.php","line":100,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"heartbeat","class":"OCA\\UserStatus\\Controller\\HeartbeatController","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/famna.de/nextcloud/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/famna.de/nextcloud/ocs/v1.php","line":65,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/famna.de/nextcloud/ocs/v2.php","line":23,"args":["/var/www/famna.de/nextcloud/ocs/v1.php"],"function":"require_once"}],"File":"/var/www/famna.de/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","Line":130}}},"CustomMessage":"Exception thrown: OC\\DB\\Exceptions\\DbalException"}}
{"reqId":"bv5Iy91TvnMZzo7Y6p32","level":2,"time":"2023-12-22T18:43:40+00:00","remoteAddr":"79.222.238.167","user":"01976b91-c1e3-4fe2-9601-66e56ac766d3","app":"no app in context","method":"GET","url":"/settings/admin/groupware","message":"Invalid google_oauth_client_id data provided to provideInitialState by mail","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/119.0","version":"28.0.1.1","data":[]}
{"reqId":"bv5Iy91TvnMZzo7Y6p32","level":2,"time":"2023-12-22T18:43:40+00:00","remoteAddr":"79.222.238.167","user":"01976b91-c1e3-4fe2-9601-66e56ac766d3","app":"no app in context","method":"GET","url":"/settings/admin/groupware","message":"Invalid microsoft_oauth_client_id data provided to provideInitialState by mail","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/119.0","version":"28.0.1.1","data":[]}
{"reqId":"ONa0ZkkAIBpdcHkqaGvs","level":2,"time":"2023-12-22T18:45:09+00:00","remoteAddr":"79.222.238.167","user":"01976b91-c1e3-4fe2-9601-66e56ac766d3","app":"no app in context","method":"GET","url":"/settings/admin/groupware","message":"Invalid google_oauth_client_id data provided to provideInitialState by mail","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/119.0","version":"28.0.1.1","data":[]}
{"reqId":"ONa0ZkkAIBpdcHkqaGvs","level":2,"time":"2023-12-22T18:45:09+00:00","remoteAddr":"79.222.238.167","user":"01976b91-c1e3-4fe2-9601-66e56ac766d3","app":"no app in context","method":"GET","url":"/settings/admin/groupware","message":"Invalid microsoft_oauth_client_id data provided to provideInitialState by mail","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/119.0","version":"28.0.1.1","data":[]}
```

**Comment by joshtrichards** [comment]
Hi @nagmat84 - I'm unable to test this at the moment since I don't have an ldap server on-line at hand.

For this and #42475 can you:

- confirm the same behavior from a different browser (preferably also without any extensions active)
- check your browser Console tab for any clues?

**Comment by nagmat84** [comment]
Sorry, I forgot to put that into the report int the first place: Tested with Firefox 119.0.1 and Chrome 119.0.6045.123 on Gentoo Linux and Firefox 121.0 on Windows 11 (each with all extensions disabled).

The only suspicious message in the developer console (in Firefox and Chrome) is: `Proxying an event bus of version 3.1.0 with 1.3.0 index.es.js:2337:14`. But I don't believe that this is bug related as this message appears on every Nextcloud page.

**Comment by nagmat84** [comment]
I guess I have found the underlying problem for this bug as well as #42475. It seems to be a permission-related problem.

I had promoted the LDAP group `admins` to be the administrative group via `php occ ldap:promote-group admins`. If I log in with an LDAP user which is in that group, the issue at hand as well as #42475 holds.

However, if I login with the original admin user `admin` which has been created during the Nextcloud installation and which is member of the built-in group `admin` (without plural-s), then the issue at hand as well as #42475 disappears.

I gave it a trial and also made one of my LDAP-admin users (`tes03` in my case)  a member of the built-in group `admin` via `php occ group:adduser admin 01976b91-c1e3-4fe2-9601-66e56ac766d3`. Now, `php occ user:info 01976b91-c1e3-4fe2-9601-66e56ac766d3` looks like:

```
  - user_id: 01976b91-c1e3-4fe2-9601-66e56ac766d3
  - display_name: Testkonto 03 (test03@famna.de)
  - email: test03@famna.de
  - cloud_id: 01976b91-c1e3-4fe2-9601-66e56ac766d3@cloud.famna.de
  - enabled: true
  - groups:
    - admin
    - users
    - admins
  - quota: none
  - storage:
    - free: 2037256683520
    - used: 0
    - total: 2037256683520
    - relative: 0
    - quota: -3
  - last_seen: 2023-12-24T13:50:51+00:00
  - user_directory: /var/www/famna.de/nextcloud/data/01976b91-c1e3-4fe2-9601-66e56ac766d3
  - backend: LDAP
```

Please note, that my user `test03` is now member of `admins` (the LDAP admin group) and `admin` (the NC built-in group). After I logged out with `test03` and logged in again, `test03` the issues have vanished.

Apparently, promoting the LDAP group `admins` to have admin privileges is not sufficient.

**Comment by come-nc** [comment]
@nagmat84 So can this and #42475 be closed since they are duplicates of #42480 from what I understood?

**Comment by nextcloud-command** [comment]
This issue has been automatically marked as stale because it has not had recent activity and seems to be missing some essential information. It will be closed if no further activity occurs. Thank you for your contributions.
